lib/rubocop/cop/sorbet/signatures/enforce_signatures.rb

```diff
@@ -63,22 +71,23 @@ def autocorrect(node)
           end
         end
 
+        def scope(node)
+          return nil unless node.parent
+          return node.parent if [:begin, :class, :module].include?(node.parent.type)
```

Comment:
What about blocks?

Reply:
Good catch, I was storing the sig in the block's parent scope!